stdlib/Std/snippets.yaml

```diff
@@ -6,39 +6,39 @@ Std.Base:
       documentation: Creates a map with a single element with key "foo" and value 10.
     - name: JSON.empty
       documentation: Empty JSON object.
-    - name: readFile $ Package.samples "Std" / "exampleFile.txt"
+    - name: readFile $ Package.samples "Base" / "exampleFile.txt"
```

Comment:
Are you certain that we want this version of the line? I believe package should take package name, being the library name, being `Std` in this case.

Reply:
true